src/Dfc.CourseDirectory.Core/DataStore/Sql/Queries/GetLearningAimRefAndEffectiveTo.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Models;
+
+namespace Dfc.CourseDirectory.Core.DataStore.Sql.Queries
+{
+    // Return all LearningAimRefs 
+    public class GetLearningAimRefAndEffectiveTo : ISqlQuery<Lars>
+    {
+        public Lars Lars { get; set; }
+
+        public string LearningAimRef { get; set; }
+    }
```

Comment:
This should query all the codes in one go
```suggestion
    public class GetLearningDeliveries : ISqlQuery<IReadOnlyDictionary<string, LearningDelivery>>
    {
        public IEnumerable<string> LearnAimRefs { get; set; }
    }
```